FirebaseAuth/Tests/Unit/SwiftAPI.swift

```diff
@@ -262,23 +262,23 @@ class AuthAPI_hOnlyTests: XCTestCase {
   #if !os(macOS)
     func FIRFedederatedAuthProvider_h() {
       class FederatedAuthImplementation: NSObject, FederatedAuthProvider {
+        @available(iOS 13, tvOS 13, macOS 10.15, macCatalyst 13, watchOS 7, *)
```

Comment:
Is this file's diff reflective of the API surface in `master` or has there been a divergence?

Reply:
This specific change is because with the Swift implementation, the async version of the API needs to be also implemented. I'll add another note to the spreadsheet to track the open question of how to deal with public protocols with previously auto-generated async apis.

There are a few other divergences because auth-swift ran these tests on mac for the first time and also, Swift is strict about assignments when there is not a `@discardableResult`

Reply:
Understood-- thanks 👍